index.js

```diff
@@ -0,0 +1,4 @@
+var path = require('path');
+module.exports = {
+	'dirname': path.join(__dirname,'template_src')
```

Comment:
A few syntax formatting suggestions:

- empty newline between L1 and L2
- replace tab with four spaces on L3
- change `'dirname'` to `dirname`
- add a space after the comma on L3: `path.join(__dirname, 'template_src')`
- add `;` on L4

Most of these suggestions are just based on the style that I see in the JavaScript used in the project.